plugins/modules/azure_rm_storageaccount.py

```diff
@@ -94,6 +94,50 @@
             -  Allows https traffic only to storage service when set to C(true).
         type: bool
         version_added: "2.8"
+    network_acls:
+        description:
+            - Manages the Firewall and virtual networks settings of the storage account.
+        type: dict
+        version_added: "2.10"
+        suboptions:
+            default_action:
+                description:
+                    - Default firewall traffic rule.
+                    - If set to 'Allow' no other settings have effect.
+                choices:
+                    - Allow
+                    - Deny
+                default: Allow
+            bypass:
+                description:
+                    - When C(default_action) is set to 'Deny' this controls which Azure components can still reach the Storage Account.
+                    - The list is comma separated.
+                    - It can be any combination of the following: AzureServices, Logging, Metrics.
+                    - If no Azure components are allowed, explicitly set C(bypass="").
+                default: AzureServices
+                suboptions:
+                    virtual_network_rules:
+                        description:
+                            - A list of subnets and their actions
+                        suboptions:
+                            id:
+                                description:
+                                    - The complete path to the subnet.
+                            action:
+                                description:
+                                    - The only logical action is 'Allow' because this setting is only accessible when C(default_action) is set to 'Deny'.
+                                default: 'Allow'
+                    ip_rules:
+                        description:
+                            - A list of ip addresses or ranges in CIDR format
+                        suboptions:
+                            value:
+                                description:
+                                    - The ip address or range.
+                            action:
+                                description:
+                                    - The only logical action is 'Allow' because this setting is only accessible when C(default_action) is set to 'Deny'.
```

Comment:
```suggestion
                                    - The only logical I(action=Allow) because this setting is only accessible when I(default_action=Deny).
```